From 83f71fbc66fbce7b93b13b64105315bfb1f70605 Mon Sep 17 00:00:00 2001
From: Akira Yokosawa <akiyks@gmail.com>
Date: Thu, 2 Jul 2026 19:44:15 +0900
Subject: [PATCH 04938/15333] docs: submitting-patches: Fix section structure
 around DCO

Commit 5903019b2a5e ("Documentation/SubmittingPatches: convert it to
ReST markup") made "DCO 1.1" a subsection under the section "Sign your
work - ...".  However, the DCO text is meant to be referenced in the
section.

Furthermore, in HTML and PDF docs, paragraphs following the DCO:

    then you just add a line saying:

        Signed-off-by: [...]

    [...]
    as it was propagated to the maintainers and ultimately to Linus, with
    the first SoB entry signalling primary authorship of a single author.

appear to belong to the DCO, rather than to "Sign your work - ..."
where it should.

In reST (and other documentation tools), once you start a subsection
under a section, there is no way to go back to the section.

Fix the section structure by making the DCO a literal block, with
indent changes made in commit 5903019b2a5e reverted.

While at it, fix the same issue at "Reviewer's statement of
oversight".

Signed-off-by: Akira Yokosawa <akiyks@gmail.com>
Tested-by: Randy Dunlap <rdunlap@infradead.org>
Acked-by: Randy Dunlap <rdunlap@infradead.org>
Signed-off-by: Jonathan Corbet <corbet@lwn.net>
Message-ID: <5f320260-5cb0-42cc-949a-c92404124618@gmail.com>
---
 Documentation/process/submitting-patches.rst | 14 ++++++--------
 1 file changed, 6 insertions(+), 8 deletions(-)

diff --git a/Documentation/process/submitting-patches.rst b/Documentation/process/submitting-patches.rst
index cc6a1f73d7f2..7ae79452e1b4 100644
--- a/Documentation/process/submitting-patches.rst
+++ b/Documentation/process/submitting-patches.rst
@@ -404,12 +404,11 @@ patches that are being emailed around.
 The sign-off is a simple line at the end of the explanation for the
 patch, which certifies that you wrote it or otherwise have the right to
 pass it on as an open-source patch.  The rules are pretty simple: if you
-can certify the below:
+can certify the below::
 
-Developer's Certificate of Origin 1.1
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+        Developer's Certificate of Origin 1.1
 
-By making a contribution to this project, I certify that:
+        By making a contribution to this project, I certify that:
 
         (a) The contribution was created in whole or in part by me and I
             have the right to submit it under the open source license
@@ -554,12 +553,11 @@ some testing has been performed, provides a means to locate testers for
 future patches, and ensures credit for the testers.
 
 Reviewed-by:, instead, indicates that the patch has been reviewed and found
-acceptable according to the Reviewer's Statement:
+acceptable according to the Reviewer's Statement::
 
-Reviewer's statement of oversight
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+	Reviewer's statement of oversight
 
-By offering my Reviewed-by: tag, I state that:
+	By offering my Reviewed-by: tag, I state that:
 
 	 (a) I have carried out a technical review of this patch to
 	     evaluate its appropriateness and readiness for inclusion into
-- 
2.55.0

